app/models/user.rb

```diff
@@ -96,10 +97,11 @@ class User < ActiveRecord::Base
 
   has_many :notifications, dependent: :destroy
   has_many :comments, dependent: :destroy
-  has_many :attachments
+  has_many :attachments, dependent: :destroy
```

Comment:
this could wreck documents and seems strange considering you're leaving comments?